### DISCUSSION TITLE
How many operations a REST API can do?

### DISCUSSION THREAD

**Discussion by Khalid-Walid12** [community/community#142464] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

How many operations a REST API can do? And what are these?

**Comment by webdevmahdi** [comment]
Mostly we run 4 operations with REST APIs called CRUD. But there are more operations available for rest API. There are 12 types of operation in REST API.

**Reply by rov1dev** [reply]
mnogo

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/142452#discussioncomment-11042211) for reference to why this discussion was closed and locked. Thanks!
